examples/cookbook/src/lib.rs

```diff
@@ -236,15 +233,15 @@ mod tests {
         assert_eq!(blocks.len(), 3);
         assert_eq!(
             *wallet.get_balances().await?.iter().next().unwrap().1,
-            225883
+            1000000000
         );
         assert_eq!(
             *wallet.get_balances().await?.iter().next().unwrap().1,
-            225883
+            1000000000
```

Comment:
I would import the constant instead of hardcoding a large value here

Reply:
done